plugins/food_bot_plugin.py

```diff
@@ -156,6 +166,15 @@ def check_rating(rating_val):
             return True
 
     @staticmethod
+    def check_rating_time(meal, now=datetime.now().strftime('%H:%M:%S')):
+        breakfast_time = Helper.get_meal_time('breakfast')
+        lunchtime = Helper.get_meal_time('lunch')
+        if (meal == 'breakfast' and now > breakfast_time) or (meal == 'lunch' and now > lunchtime):
+            return True
```

Comment:
@andela-ydaniju Usually you do not return True or False in statements like this. You just return the comparism and python handles the result of True of False based on what is given.